hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/trigger/TriggerAction.java

```diff
@@ -40,26 +43,55 @@
  * @author Jay Shaughnessy
  * @author Lucas Ponce
  */
+@ApiModel(description = "Link an <<ActionDefinition>> with a <<Trigger>>. + \n" +
+        " + \n" +
+        "It can add optional constraints that determine when an action will be executed: + \n" +
+        " + \n" +
+        "- A set of Alert.Status (represented by its string value). + \n" +
+        "The action will be executed if the Alert which is linked is on one of the states defined. + \n" +
```

Comment:
Suggestion:
'The action will be executed if the Alert which is linked is on one of the states defined. '
-->
'The action will be executed if the linked Alert is set to one of the defined states. '